index.bs

```diff
@@ -371,6 +374,28 @@ To <dfn abstract-op>initialize a global object's embedder policy from a response
 
 </div>
 
+### Process a navigation fetch ### {#process-a-navigation-fetch}
+<div algorithm="process a navigation fetch">
+For every navigation request for a nested browsing context whose parent's embedder
+policy is "`require-corp`", CORP is checked.
```

Comment:
You'll want to add this to the list of changes in https://mikewest.github.io/corpp/#integration-html (around line 333). It might be reasonable to drop this whole section in favor of simply monkey-patching there, or it may make sense to label the following as an algorithm that's called-out-to in the same way we do for "process a navigate response". I'm happy either way. Depending on which route you take, I'd adjust this text accordingly, perhaps copy/pasting the existing "process a navigation response" text, a la:

```suggestion
If a document's [=document/embedder policy=] is "`require-corp`", then any document it embeds in a
[=nested browsing context=] must positively assert a "`require-corp`" [=/embedder policy=] (see
[[#cascade-vs-require]]). Here, we modify HTML's [=process a navigation fetch=] algorithm to perform
the CORP check upon each redirect in a navigation chain.
```

Reply:
I inlined the section. Does it look good?